lib/features/local_live_log.js

```diff
@@ -1,9 +1,3 @@
-/**
```

Comment:
This module handles the creation and exposure of a live log endpoint useful for seeing a stream of the task output while the task is running.  Upon completion of the task, the live log will terminate and be redirected to a file artifact with the entirety of the task output.